src/tango/validator/fd_leader_schedule.c

```diff
@@ -0,0 +1,32 @@
+#include "fd_leader_schedule.h"
+#include "../../util/wksp/fd_wksp_private.h"
+#include <stdio.h>
+
+void *
+fd_leader_schedule_new( void * mem )
+{
+  fd_leader_schedule_t * schedule = ( fd_leader_schedule_t * )mem;
+  fd_memset( schedule, 0, fd_leader_schedule_footprint() );
+  return (void *)schedule;
+}
+
+/* TODO: LML same functionality as run.c:workspace_pod_join we should refactor */
+fd_leader_schedule_t *
+fd_leader_schedule_get( char const * app_name ) {
```

Comment:
I don't think this belongs in tango, the layout of the pod at `gaddr_lo` is a higher level thing that belongs more to fdctl or the tile, than tango.

Reply:
It doesn't belong but this code won't be around for long - once we land https://github.com/firedancer-io/firedancer/blob/mmcgee/solanabank2/ffi/rust/firedancer/src/pod.rs#L44 I'll use that on the rust side and remove this function entirely.